src/components/Lists/index.js

```diff
@@ -112,6 +122,11 @@ function Lists({ results: listsProp, paginationProps, ...props }) {
           accountsLabel="User Accounts"
           accountsOnChange={handleChange}
           accountsHelper="Enter twitter account names seperated by a comma i.e userone,usertwo"
+          accountsErrorHelper={
+            !isLoading && isError
+              ? "Please enter a valid twitter account/s and that multiple accounts are separated by a comma."
```

Comment:
```suggestion
              ? "Please enter a valid comma-separated list of Twitter usernames."
```
Or something like that...